static/app/views/starfish/views/webServiceView/spanGroupBreakdown.tsx

```diff
@@ -71,7 +76,8 @@ export function SpanGroupBreakdown({
           data={visibleSeries}
           start=""
           end=""
-          loading={false}
+          errored={errored}
+          loading={isTimeseriesLoading}
```

Comment:
do we need to also check `isCumulativeTimeLoading` here? just wondering because it was added to the component props and it looks unused right now.

Reply:
yeah you're right, it's unused, will clean it in a follow up so i can merge this!